### ISSUE TITLE
model 관련해서 질문드립니다.

### ISSUE THREAD

**Issue by MunJeongHyeon** [openspeech-team/openspeech#103]
# ❓ Questions & Help

모델 정리해놓으신 readme 파일에서 17번 19번이 conformer 모델인데
conformer_lstm이 17번과 19번 모두에 해당되는것이 맞을까요?

**Comment by MunJeongHyeon** [comment]
또한 제가 현재 모델을 돌렸는데 
![image](https://user-images.githubusercontent.com/77957228/133909233-3272b5fb-456f-424f-af97-469e9392d0b2.png)

위와 같은 에러가 발생합니다. 좀 찾아보았지만 해결이 잘 안 돼서 이렇게 질문 남깁니다...

**Comment by sooftware** [comment]
conformer 페이퍼의 모델의 경우 lstm 디코더가 아닌 트랜스듀서 디코더 모델입니다.   
에러의 경우 조금 더 자세한 정보를 남겨주세요.

**Comment by MunJeongHyeon** [comment]
제가 모델을 conformer_lstm이 아닌 conformer로 돌렸을 때 디코더의 모델이 없어서 모델이 안돌아갔습니다. 혹시 conformer_lstm 모델말고 readme 파일의 17번째 conformer 모델의 model 이름은 커맨드창에 뭐라고 작성을 해야할까요?

**Comment by MunJeongHyeon** [comment]
에러의 자세한 정보를 어떻게 남기면 될까요???
제가 돌린 모델은 conformer_lstm 모델이며 에폭이 어느정도 돌아가고 cer이 잘찍히다가 갑자기
RuntimeError: cuDNN error: CUDNN_STATUS_EXECUTION_FAILED 라는 에러가 발생하고 학습이 끝나게 됩니다.

**Comment by hasangchun** [comment]
17번째 [conformer](https://arxiv.org/abs/2005.08100) 모델의 경우 트랜스듀서 모델이기 때문에 모델 이름은 `conformer_transducer` 이고, criterion은 `transducer`입니다. [(관련 링크)](https://github.com/openspeech-team/openspeech/blob/main/openspeech/models/conformer/model.py#L221-L269)

Environment info랑 에러 로그 같이 복붙해서 올려주시면 좋을 것 같아요~!

**Comment by hasangchun** [comment]
네, transducer 모델은 joint 계산 후 4차원으로 나와서 cer이나 wer을 계산하지 않는 것으로 구현되어 있습니다.

**Comment by arabae** [comment]
@MunJeongHyeon
openspeech/utils.py 파일에 num_devices가 2개로 잘 잡고 있을까요? 저는 이 부분을 "num_devices=2" 이렇게 명시적으로 바꿔주니 잘 동작했습니다!

**Comment by sooftware** [comment]
트랜스듀서 모델은 메모리가 4차원으로 잡혀서 많이 필요합니다 ㅎㅎ..

**Comment by sooftware** [comment]
LSTM의 경우 트랜스듀서 모델이 아니라 메모리가 훨씬 적게 먹긴 합니다만, conformer 인코더에서 메모리를 꽤 먹습니다 ㅎㅎ

**Comment by MunJeongHyeon** [comment]
넵!!! 좋은답변 정말 감사합니다!!!!! 늘 언제나 감사합니다.!!!
